src/components/accessories/opds/filter/OpdFilterForm.tsx

```diff
@@ -311,11 +311,11 @@ export const OpdFilterForm: FC<IOpdFilterProps> = ({
   const newPatientOptions = [
     {
       label: t("opd.all"),
-      value: "",
+      value: "A",
     },
     {
       label: t("opd.newadmittance"),
```

Comment:
can you please also change “New Admittance” into “New Attendance”?

Reply:
Sure...